service/initialise.go

```diff
@@ -80,8 +75,17 @@ func (e *Init) DoGetHealthCheck(cfg *config.Config, buildTime, gitCommit, versio
 
 // DoGetMongoDB returns a MongoDB
 func (e *Init) DoGetMongoDB(ctx context.Context, cfg *config.Config) (api.PermissionsStore, error) {
-	mongodb := &mongo.Mongo{}
-	if err := mongodb.Init(ctx, shouldEnableReadConcern, shouldEnableWriteConcern); err != nil {
+	mongodb := &mongo.Mongo{
+		Collection:              cfg.MongoConfig.Collection,
```

Comment:
Just a thought.. I think it's good for packages to have their own config structs anyway, but seems like it could be especially appropriate here.. If the config package has a specific 'mongo config' struct, would it make sense to move that config struct to the mongo package itself? You can just pass the whole mongo.Config to the mong.Init function straight from the config, initialise everything there and would save having to do it in each service. Maybe it would force too many changes for now across the services that use it but would definitely be an idea to think about